labs-ios-starter.xcodeproj/project.pbxproj

```diff
@@ -318,7 +318,7 @@
 /* Begin PBXShellScriptBuildPhase section */
 		922C5AE624F82DB300A28927 /* ShellScript */ = {
 			isa = PBXShellScriptBuildPhase;
-			buildActionMask = 12;
+			buildActionMask = 8;
```

Comment:
Revert this file. These changes were caused by checking "Run script only when installing" in the Swift Lint run script.